ui/library/test/utils/scrollDirectionDetector.test.ts

```diff
@@ -0,0 +1,194 @@
+import { expect } from 'chai';
+
+import ScrollDetector from '../../../library/src/utils/ScrollDirectionDetector';
+import { Nullable } from '../../src/components/types/utils';
+import { ScrollDirection } from '../../src/context/ScrollContext';
+
+describe('scrollDirectionDetector', () => {
+  it.only('should get a default null value for Direction', () => {
```

Comment:
Don't commit `.only` on PRs. That's why the `scholar` repo will break `yarn unit:test` when there are `.only()`'s in the code. `yarn testOnly` allows them, because that is for individual test debugging. Might be worth adding a similar thing in this repo.